Jenkinsfiles/maint-mode.yml

```diff
@@ -0,0 +1,3 @@
+pipeline:
```

Comment:
@jwhitlock I object because I think it should not be in master. After merging, this file will also be in master that It should not!